code/ai/aicode.cpp

```diff
@@ -7888,11 +8030,13 @@ void update_aspect_lock_information(ai_info *aip, vec3d *vec_to_enemy, float dis
 			(tshpp == NULL ||
 			ship_get_closest_subsys_in_sight(tshpp, SUBSYSTEM_ENGINE, &aiobjp->pos))))) {
 				aip->aspect_locked_time += flFrametime;
-				if (aip->aspect_locked_time >= wip->min_lock_time) {
-					aip->aspect_locked_time = wip->min_lock_time;
```

Comment:
Hmm, why remove this line?

Reply:
Normally, `aspect_locked_time` is capped at `min_lock_time`, but for multilock, it needs to go above so it can get 2x 3x, etc for double-locking, triple-locking etc.

The only other time this is relevant is when lock is lost, where it would've normally started counting down from `min_lock_time`, since it was capped there, so now the lines below were added to reset it back to `min_lock_time` if it was above.